src/scenario/scenario.c

```diff
@@ -7,36 +7,124 @@
 #include "game/save_version.h"
 #include "game/settings.h"
 #include "scenario/data.h"
+#include "scenario/request.h"
 
 struct scenario_t scenario;
 
+static struct {
+    int size;
+    int start_info;
+    int old_filler_1;
+    int invasions_part1;
+    int funds_enemy;
```

Comment:
This is an ambiguous name. It seems to read "enemy funds", but it's actually city funds and enemy id. I think this needs better naming.

Reply:
Done.